CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+## Commands to know:
+
+* `yarn run dev` builds and starts the development **web** server.
+* `yarn run dev` builds and starts the production **web** server.
```

Comment:
Is this right?

Reply:
Nope. Should be `yarn run prod`. Thanks for catching, I'll fix.